pkg/chartrenderer/default.go

```diff
@@ -160,31 +160,48 @@ func (c *RenderedChart) Manifest() []byte {
 }
 
 // Files returns all rendered manifests mapping their names to their content.
```

Comment:
Docstring has to be updated, it is no longer the mapping manifest name to content.